editor/editor_help.cpp

```diff
@@ -2620,6 +2621,166 @@ void EditorHelpBit::_meta_clicked(String p_select) {
 	}
 }
 
+String EditorHelpBit::get_class_description(const StringName &p_class_name) const {
+	HashMap<String, DocData::ClassDoc>::ConstIterator E = EditorHelp::get_doc_data()->class_list.find(p_class_name);
+	if (!E) {
+		return String();
+	}
+
+	// Non-native properties shouldn't be translated.
+	bool is_native = ClassDB::class_exists(p_class_name);
+	// Caching descriptions would be overkill.
+	return is_native ? DTR(E->value.brief_description) : E->value.brief_description;
+}
```

Comment:
So class descriptions are not cached now?

Reply:
There was never consistency to if they were cached or not before. They mostly weren't, and I still think it's overkill to do so, since we don't need loops in order to find them.

Reply:
It would avoid going through a loop in `ClassDB::class_exists` and all the checks and loops of `DTR` though.